x11-wm/compiz/metadata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<use>
```

Comment:
Pleas put `<use/>` below maintainers; when we open this file we usually need to read maintainers, so it's best to keep them on top.